Projects/HomeWork2/HomeWork/HomeWork/Program.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+
+namespace HomeWork
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter task number:");
+            int taskN = Int32.Parse(Console.ReadLine());
+            if (taskN == 201)
+            {
+                Console.WriteLine("Enter first number");
+                int a = Int32.Parse(Console.ReadLine());
+                Console.WriteLine("Enter second number");
+                int b = Int32.Parse(Console.ReadLine());
+                Console.WriteLine("Result=" + (a + b) / 2.0);
+            }
+            else if (taskN == 203)
+            {
+                Console.WriteLine("Enter number");
+                int a = Int32.Parse(Console.ReadLine());
+                if (a % 2 == 0) Console.WriteLine("parne");
+                else Console.WriteLine("ne parne");
+            }
+            else if (taskN == 204)
+            {
+
+                Console.WriteLine("Enter number");
+                int a = Int32.Parse(Console.ReadLine());
+                int count = 0;
+                int sum = 0;
+                while (a > 0)
+                {
+                    count++;
+                    sum += a % 10;
+                    a = a / 10;
+                }
+                Console.WriteLine("Count of numbers: " + count + " Sum of numbers: " + sum);
+            }
+            else if (taskN == 205)
+            {
+
+                Console.WriteLine("Enter number");
+                int a = Int32.Parse(Console.ReadLine());
+                int num = 0;
+                while (a > 0)
+                {
+                    num = a % 10;
+                    a = a / 10;
+                    Console.Write(num + " ");
+                }
+
+
+            }
+            else if (taskN == 206)
+            {
+
+                Console.WriteLine("Enter number");
+                int a = Int32.Parse(Console.ReadLine());
+                int sum = 0;
+                while (a > 0)
+                {
+                    sum += a % 10;
+                    a = a / 10;
+                }
+                Console.WriteLine(" Sum of numbers: " + sum);
+
+            }
+            else if (taskN == 207)
+            {
+                Console.WriteLine("Same element: ");
+                int[] a = new int[6] { 2, 2, 4, 5, 6, 6 };
+
+                Console.WriteLine();
+                for (int i = 0; i < a.Length; i++)
+                {
+                    for (int j = i + 1; j < a.Length; j++)
+                    {
+                        if (a[i] == a[j])
+                        {
+                            Console.WriteLine(a[i] + " ");
+                        }
+                    }
+                }
+            }
+            else if (taskN == 208)
+            {
+                Console.WriteLine(" Min value: ");
+                int[] a = new int[9] { 2, 2, 4, 6, 4, 8, 6, 5, 10};
+                int min = 100000;
```

Comment:
will your code work for this array { 1000001, 1000002, 1000003, 1000003, 1000003, 1000003, 1000003, 1000003, 1000003 } ??
:)

could you please improve it a bit?

Reply:
it is better to avoid min or max values in our algorithms because it causes some limitations or even errors.

This particular algorithm can work without "int min =0 ".
I believe that it is better just to start from the first array value "int min = a[0]";